provisioning/dashboards/datasource.json

```diff
@@ -167,6 +167,350 @@
+            "id": "maxArea",
+            "labelWidth": 10,
+            "rows": 10,
+            "section": "",
+            "title": "maxArea",
+            "tooltip": "",
+            "type": "disabledTextarea",
+            "uid": "965c7503-f229-4549-b5df-24182e42ab1b",
+            "unit": "",
+            "value": ""
+          },
+          {
+            "id": "minArea",
+            "labelWidth": 10,
+            "rows": 10,
+            "section": "",
+            "title": "minArea",
+            "tooltip": "",
+            "type": "disabledTextarea",
+            "uid": "c9c4c436-d3e6-4f27-9d3a-91c3b9c2d5dd",
+            "unit": "",
+            "value": ""
+          }
+        ],
+        "initial": {
+          "code": "context.panel.patchFormValue({\n  \"speed\": \"15\",\n  \"max\": 220,\n  \"min\": 230,\n  \"maxArea\": 250,\n  \"minArea\": 260\n});\n",
+          "contentType": "application/json",
+          "getPayload": "return {}",
+          "highlight": false,
+          "highlightColor": "red",
+          "method": "-",
+          "payload": {}
+        },
+        "layout": {
+          "orientation": "horizontal",
+          "padding": 10,
+          "sectionVariant": "default",
+          "variant": "single"
+        },
+        "reset": {
+          "backgroundColor": "purple",
+          "foregroundColor": "yellow",
+          "icon": "process",
+          "text": "Reset",
+          "variant": "hidden"
+        },
+        "resetAction": {
+          "code": "if (context.panel.response) {\n  context.grafana.notifySuccess(['Update', 'Values updated successfully.']);\n  context.grafana.locationService.reload();\n} else {\n  context.grafana.notifyError(['Update', 'An error occurred updating values.']);\n}",
+          "confirm": false,
+          "getPayload": "return {}",
+          "mode": "initial",
+          "payload": {}
+        },
+        "saveDefault": {
+          "icon": "save",
+          "text": "Save Default",
+          "variant": "hidden"
+        },
+        "submit": {
+          "backgroundColor": "purple",
+          "foregroundColor": "yellow",
+          "icon": "cloud-upload",
+          "text": "Submit",
+          "variant": "primary"
+        },
+        "sync": true,
+        "update": {
+          "code": "if (context.panel.response && context.panel.response.state === 'Done') {\n  context.grafana.notifySuccess(['Update', 'Values updated successfully.']);\n  context.panel.initialRequest();\n} else {\n  context.grafana.notifyError(['Update', 'An error occurred updating values.']);\n}",
+          "confirm": true,
+          "contentType": "application/json",
+          "datasource": "PostgreSQL",
+          "getPayload": "const payload = {};\n\ncontext.panel.elements.forEach((element) => {\n  if (!element.value) {\n    return;\n  }\n\n  payload[element.id] = element.value;\n})\n\n/**\n * Data Source payload\n */\nreturn payload;",
+          "method": "datasource",
+          "payload": {
```

Comment:
I think it should be normalized to an empty object

Reply:
updated